### PR TITLE
Add indentation to stream pane

### PR DESCRIPTION
**Pull request by RonSkons**
See [previous PR](https://github.com/itext/i7j-rups/pull/83).
Squashed commits to [improve readability](https://github.com/itext/i7j-rups/pull/83#issuecomment-1602467584).

### CODE REVIEW

**Comment by MatthiasValvekens** [comment]
Just as an FYI for future reference: you can force-push squashed/rebased commits to GitHub PR branches without creating a new PR; GitHub is smart enough to figure out how to deal with those :)

**Comment by RonSkons** [comment]
> Just as an FYI for future reference: you can force-push squashed/rebased commits to GitHub PR branches without creating a new PR; GitHub is smart enough to figure out how to deal with those :)

Good to know. When I renamed the branch, it automagically closed the first PR.

**Review by Minothor — APPROVED** [review]
Looks good, Cheers Ron!

File: src/test/java/com/itextpdf/rups/view/itext/contentstream/StyledSyntaxDocumentTest.java [hunk 1]
```diff
@@ -611,4 +611,59 @@ public void testNoTooltip() throws Exception {
...
     }
+
+    @Test
+    public void testIndentBaseline() throws Exception {
+        byte[] origBytes = Files.readAllBytes(Paths.get(SRC_DIR, "baseline.cmp"));
+        StyledSyntaxDocument doc = new StyledSyntaxDocument();
+        doc.processContentStream(origBytes);
+
+        String output = doc.getText(0, doc.getLength());
+        String expectedResult = new String(Files.readAllBytes(Paths.get(SRC_DIR, "baselineIndented.cmp")));
```
Comment (Minothor) [inline src/test/java/com/itextpdf/rups/view/itext/contentstream/StyledSyntaxDocumentTest.java]: @RonSkons Just a head's up that it's failing the CI quality gate at the moment due to new String(byte[]...) not having the character encoding set for this and the other 4 test cases.
If you could add that in, it should pass.

Comment (RonSkons) [reply src/test/java/com/itextpdf/rups/view/itext/contentstream/StyledSyntaxDocumentTest.java]: Should be fixed now.

**Review by Minothor — APPROVED** [review]

**Pull request merged**